### ISSUE TITLE
Fix broken hyperlink in the documentation

### ISSUE THREAD

**Issue by rakeshgowerneni** [capitalone/DataProfiler#972]
**Please provide the issue you face regarding the documentation**
The hyperlink in the below sentence in the `Load a File` section should change from

If the file type is not automatically identified (rare), you can specify them specifically, see section [Specifying a Filetype or Delimiter](https://github.com/capitalone/DataProfiler#specifying-a-filetype-or-delimiter).

to 

If the file type is not automatically identified (rare), you can specify them specifically, see section [Specifying a Filetype or Delimiter](https://capitalone.github.io/DataProfiler/docs/0.10.1/html/profiler.html?highlight=filetype#specifying-a-filetype-or-delimiter).

I am trying to fix the documentation as a way to create my first PR on the repo.

**Comment by JGSweets** [comment]
@rakeshgowerneni  that's a good catch! I'm curious why sphinx did not generate that properly. Thoughts on how to fix? We welcome PRs and would love to see you contribute!

**Comment by taylorfturner** [comment]
When you implement the fix @rakeshgowerneni, be sure its not specifying a specific version in the hyperlink. Rather, it should be agnostic to the version and just resolve to the proper location within whatever docs are live at that time. 

```If the file type is not automatically identified (rare), you can specify them specifically, see section [Specifying a Filetype or Delimiter](https://capitalone.github.io/DataProfiler/docs/0.10.1/html/profiler.html?highlight=filetype#specifying-a-filetype-or-delimiter).```

**Comment by thebadcoder96** [comment]
Hey is anyone working on this? Can I try this out?

**Comment by taylorfturner** [comment]
Yes, you are more than welcome! README.md in the `dev-gh-pages` would be a
good place to start for building documentation. Thanks and let me know if
you have any other questions. Cheers!

**Comment by taylorfturner** [comment]
@thebadcoder96 let me know if you have anymore questions -- the team is back at it today. Thanks for your interest!

**Comment by thebadcoder96** [comment]
Thank you for the response! I will go through the readme file that you mentioned and let you know if I have any questions.

**Comment by taylorfturner** [comment]
> Thank you for the response! I will go through the readme file that you mentioned and let you know if I have any questions.

Thanks @thebadcoder96!

**Comment by thebadcoder96** [comment]
Here is how I think we can solve the issue. I do not have much idea on sphinx but I am looking into that as well.

- We can run a Python script that prints out the current/live version of Data Profiler.
- Trigger a custom GitHub Actions workflow when there is a push to main and use the Python script to update the readme file using the latest version. Probably use `sed` for updating the readme file. 

Wdyt? Any other thoughts are welcome since I do not have full context on how everything works within the Data Profiler, I might be wrong as well.

**Comment by taylorfturner** [comment]
I think we can actually re-use the code written out [here](https://github.com/capitalone/DataProfiler/blob/gh-pages/docs/update_documentation.py#L79). 

That script will have the proper version tag and we can simply find the spot in the pertinent file to ensure its always referencing the most up-to-date build of the docs.

**Comment by thebadcoder96** [comment]
Please correct me if I am wrong, but the issue mentioned here pertains to the readme file of the [Data Profiler Github](https://github.com/capitalone/DataProfiler?tab=readme-ov-file#get-started). In the [documentation website](https://capitalone.github.io/DataProfiler/docs/0.10.1/html/index.html#load-a-file) there is no hyperlink and the sentence is also a bit different. 

From my understanding, `docs/update_documentation.py` updates the GitHub pages documentation, which is NOT the one we want to update right?

**Comment by taylorfturner** [comment]
@thebadcoder96 yes, you are correct. I was confusing something else.  


In this case I would key off of the `version.py` file and try to reference that in the README.md as a consistent reference point for the proper version for the hyperlink... there might be another way too to reference the `index.html` and tack on a `#specifying-a-hyoerlink`(or something like that) to the hyperlink in the README

**Comment by taylorfturner** [comment]
@thebadcoder96 the URL should be 

`https://capitalone.github.io/DataProfiler/docs/<current_version_tag>/html/profiler.html?highlight=filetype#specifying-a-filetype-or-delimiter`

We'll want the version tag to be a reference and not a manual change every time we make a new release of dataprofiler.

**Comment by thebadcoder96** [comment]
Since you mentioned version tag, I think we can use `git describe --tags` to get the current version tag and then update the readme file. 

I have created a PR with the proposed changes, please review #1080
